src/pages/Detail/DetailIntroduction.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function DetailIntroduction() {
+  return (
+    <IntroductionWrapper>
+      <IntroHeader>
+        <CategoryTitle>
+          <span>category</span>
+        </CategoryTitle>
+        <ProjectTitle>
+          <h1>유기동물 입양 문화를 만드는 포인핸드 매거진 8호</h1>
+        </ProjectTitle>
+        <CreatorProfile>
+          <img alt="profileImg" src="/images/example2.jpg" />
+          <p>creator name</p>
+        </CreatorProfile>
+      </IntroHeader>
+      <IntroImage alt="introImage" src="/images/example2.jpg" />
+      <IntroAside>
+        <ProjectStatus>
+          <CollectedAmount>
+            <p>모인금액</p>
+            <h2>3,505,000 원</h2>
+          </CollectedAmount>
+          <LeftDate>
+            <p>남은시간</p>
+            <h2>8일</h2>
+          </LeftDate>
+          <Sponsor>
+            <p>후원자</p>
+            <h2>64명</h2>
+          </Sponsor>
+        </ProjectStatus>
+        <ProjectButtons>
+          <PushThisProjectBtn>
+            <button>프로젝트 밀어주기</button>
+          </PushThisProjectBtn>
+          <LikeBtn>
+            <button>
+              <i className="far fa-heart"></i>
+            </button>
+          </LikeBtn>
+          <ShareBtn>
+            <button>
+              <i className="fas fa-share-alt"></i>
+            </button>
+          </ShareBtn>
+        </ProjectButtons>
+      </IntroAside>
+    </IntroductionWrapper>
+  );
+}
+
+const IntroductionWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  padding-bottom: 1rem;
+  line-height: 1.7em;
+  word-break: break-all;
+
+  ${({ theme }) => theme.desktop`
+  justify-content:center;
+  `};
+`;
+
+const IntroHeader = styled.header`
+  order: 2;
+  display: block;
+  padding: 1.75rem 1rem;
+  text-align: left;
+  line-height: 1.7em;
+
+  ${({ theme }) => theme.desktop`
+  order:1;
+  display:flex;
+  flex-wrap:wrap;
+  justify-content:center;
+  align-items:center;
+  width:100%;
+  height:280px;
+  `};
```

Comment:
```suggestion
  ${({ theme }) => theme.desktop`
    order: 1;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center;
    width: 100%;
    height: 280px;
  `};
```
- indent 잘 맞춰주시고
- 세로 고정폭 주기보다는 padding, margin 등으로 맞춰주는게 좋을거 같아요.

Reply:
세로폭 값 없애고 패딩값으로 조절 완료 했습니다:)